xml_inscriptions/transcribed/NJ.Princ.PU.AM.G.4740-I130a.xml

```diff
@@ -187,11 +192,11 @@ changes: Added @evidence to date
                 <lb n="9"/>εἱ μιηρὰ μέτρα <name key="Διωνυσία Dionysia">Διωνυσία</name> εἱ κὲ
                 <lb n="10"/><name key="Εἱσυχία Hesykhia">Εἱσυχία</name> ὐκῶ<supplied reason="omitted">ντα</supplied> ἐν γιτωνία Μυγδ
                 <lb n="11" break="no"/>ωνιτῶν: ὡς ἔβαλες τὼ ἅ<supplied reason="omitted">ρ</supplied>μα το
-                <lb n="12" break="no"/>ῦ Φαραῶνος, οὕτος βάλη τὶν δύσ
+                <lb n="12" break="no"/>ῦ <persName type="imperial">Φαραῶνος</persName>, οὕτος βάλη τὶν δύσ
                 <lb n="13" break="no"/><surplus>ω</surplus>ληψιν αὐτοῦ: ὦ βρ<supplied reason="omitted">ό</supplied>ντων καὶ
                 <lb n="14"/>ἀσ<surplus>σ</surplus>τρ<supplied reason="omitted">ά</supplied>πτων <name type="divine" key="Iao Ιαω">Ιαω</name>, ὡς ἐξέκωψ
-                <lb n="15" break="no"/>ης τὰ πρωτότυκα τῖς Ἠγύπ
-                <lb n="16" break="no"/>του, ἄ<supplied reason="omitted">κ</supplied>κωψων τ<unclear>ἄλο</unclear><supplied reason="lost">γα α</supplied> 
+                <lb n="15" break="no"/>ης τὰ πρωτότυκα τῖς <placeName ref="africa.egypt">Ἠγύπ
```

Comment:
I think when we're doing placeName refs in-text we'd like to start moving towards Pleiades IDs and not just our internal system.